app/design/frontend/default/default/template/mydata/mydata.phtml

```diff
@@ -1,41 +1,54 @@
 
 <?php
 $model = $this->loadByField();
-
 ?> 
-<table border='3' style='width:100%;' class="data-table cart-table" >
-    <tr class="first last">
-        <th>Profile Id</th>
-        <th>User Id </th>
-        <th>User Name</th>
-        <th>First Name</th>
-        <th>Last Name</th>
-
-    </tr>
-
-
-    <?php foreach ($model as $item) { ?>
-
-        <tr >
-            <td > <?php echo $item['profile_id']; ?></td>
-            <td > <?php echo $item['user_id']; ?></td>
-            <td> <?php echo $item['user_name']; ?></td>
-            <td> <?php echo $item['first_name']; ?></td>
-            <td> <?php echo $item['last_name']; ?></td>
-
 
+<div class="page-title">
+    <h1><?php echo Mage::helper('contacts')->__('User Information') ?></h1>
+</div>  
+
+<table class="data-table orders" id="my-orders-table">
+    <col width="1" />
+    <col width="1" />
+    <col />
+    <col width="1" />
+    <col width="1" />
+    <col width="1" />
+    <thead>
+        <tr>
+            <th class="number"><?php echo $this->__('Profile Id') ?></th>
+            <th class="date"><?php echo $this->__('User Id') ?></th>
+            <th class="ship"><?php echo $this->__('User Name') ?></th>
+            <th class="total"><span class="nobr"><?php echo $this->__('First Name') ?></span></th>
+            <th class="total"><span class="nobr"><?php echo $this->__('Last Name') ?></span></th>
+            <th class="status"><span class="nobr"><?php echo $this->__('Update') ?></span></th>
+            <th class="view"><span class="nobr"><?php echo $this->__('Dalete') ?></span></th>
         </tr>
-
-    <?php } ?>  
-
+    </thead>
+    <tbody>
+        <?php  $_odd = ''; ?>
+        <?php foreach ($model as $_order) : ?>
+            <tr >
+                <td class="number"><?php echo $_order['profile_id'] ?></td>
+                <td class="date"><span class="nobr"><?php echo $_order['user_id'] ?></span></td>
+                <td class="ship"><?php echo $_order['user_name'] ?></td>
+                <td class="total"><?php echo $_order['first_name'] ?></td>
+                <td class="status"><em><?php echo $_order['last_name'] ?></em></td>
+
+                <td> <a href="index/update?edit_id=<?php echo($_order['user_id']); ?>"  >Update</a></td>
+                <td> <a href="?id=<?php echo($_order['user_id']); ?>" class="link-reorder">Delete</a></td>
+
+            </tr>
+       <?php endforeach; ?>
+    </tbody>
 </table>
-<a  href="index/insert"> Add New Row</a>
-
-
-
-
-
-
+    
+<div class="buttons-set">
+    <p class="back-link"><a href="index/insert"><small></small>Add New Row</a></p>
+</div>
 
+<script type="text/javascript">decorateTable('my-orders-table');</script>
+<?php echo $this->getPagerHtml(); ?>

```

Comment:
sorry sir for this. i was checking , it is work or not but it is not working . now i am remove it.